Paraswap-Agents/delegate-calls/src/agent.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Finding,
+  TransactionEvent,
+  HandleTransaction,
+} from "forta-agent";
+
+import { createFinding } from "./finding";
+
+// The address of the Paraswap contract (the contract's name is "AugustusSwapper")
+const AUGUSTUS_ADDR = "0xdef171fe48cf0115b1d80b88dc8eab59176fee57";
+
+// Function signatures for `multiSwap` and `megaSwap`
+const multiPathFuncSigs = ["0xa94e78ef", "0x46c67b6d"];
```

Comment:
```suggestion
const MULTIPATH_FUNC_SIGS = ["0xa94e78ef", "0x46c67b6d"];
```
Small suggestion to make variable all caps since it is a similar variable to `AUGUSTUS_ADDR`.